kubeflow/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "0db7b333-38a1-4e09-af1b-317da2f9f413",
+  "app_id": "kubeflow",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Integration for Kubeflow",
+    "title": "Kubeflow",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Metrics",
+      "Category::Kubernetes",
```

Comment:
We should add the [AI/ML category](https://app-dev-prod.datadoghq.com/integrations?category=AI%2FML) too.